File: 200_LeetCode_Questions/Python/D01P3_LeetCode_1431.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        m=max(candies)
        l=[]
        for i in candies:
            d=i+extraCandies
            l.append(not(d<m))
        return l

File: 200_LeetCode_Questions/C++/D01P3_LeetCode_1431.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        vector<bool> res;
        int max = *max_element(candies.begin(), candies.end());
        
        for(int i=0; i < candies.size(); i++){
            if(candies[i]+extraCandies >= max)
                res.push_back(true);
            else
                res.push_back(false);
        }
        
        return res;
    }
};